nvm.sh

```diff
@@ -22,7 +22,7 @@ function setIdeaToUseNvm() {
     if [ -f .idea/workspace.xml ]; then
         NODE_VERSION=$(nvm use |  grep -oEi 'v[0-9.]+' | head -1)
         echo Setting IDEA workspace to $NODE_VERSION
-        sed -i "s/\.nvm\/versions\/node\/[^/]\+/\.nvm\/versions\/node\/${NODE_VERSION}/g" .idea/workspace.xml
+        sed -i "" "s/\.nvm\/versions\/node\/[^/]\+/\.nvm\/versions\/node\/${NODE_VERSION}/g" .idea/workspace.xml
```

Comment:
macOs' version of sed ‘-i’ flag has a [mandatory suffix(https://markhneedham.com/blog/2011/01/14/sed-sed-1-invalid-command-code-r-on-mac-os-x/)].